### PR TITLE
Content.ad: CDN domain processing

### PR DESCRIPTION
**Pull request by jlucero-contentad**
Resolve issue where CDN domain processing failed when the host did not match the requesting domain

### CODE REVIEW

**Comment by jridgewell** [comment]
to @lannka

**Comment by jlucero-contentad** [comment]
Ping @lannka

File: ads/contentad.js [hunk 1]
```diff
@@ -35,9 +35,10 @@ export function contentad(global, data) {
   /* Pass Source URL */
   let sourceUrl = window.context.sourceUrl;
   if (data.url) {
-    const host = window.context.location.host;
+    const s = document.createElement('a');
```
Comment (lannka) [inline ads/contentad.js]: you can use `parseUrl` from `url.js`

Comment (jlucero-contentad) [inline ads/contentad.js]: I thought I was using that - see lines 39 and 41 below.

Comment (jlucero-contentad) [inline ads/contentad.js]: Are you suggesting that I should use a different method @lannka?

Comment (jridgewell) [inline ads/contentad.js]: Yes, you should call `parseUrl` to replace the anchor element/`s.href` code here.

Comment (jlucero-contentad) [inline ads/contentad.js]: @jridgewell I think I got it. I updated the code to use parseUrl. Please let me know if anything further needs to be done to get this merge request approved.

**Comment by jridgewell** [comment]
Ping @lannka or @zhouyx

**Comment by jlucero-contentad** [comment]
@jridgewell any chance this pull request could go out with this week's release?

**Comment by jridgewell** [comment]
We just cut canary last night, so not this week. Additionally, we have a 2 week release schedule, so yours will go into canary next Wed, and production the following Wed.

**Comment by jlucero-contentad** [comment]
Oh ok. Previously release day was Thursdays. But next week will have to do. Thank you!
Is there any thing I can do to help get this merge approved for the next release?

**Comment by jridgewell** [comment]
> Is there any thing I can do to help get this merge approved for the next release?

@lannka

**Comment by zhouyx** [comment]
LGTM. 
Have one concern:  `parseUrl` will freeze result during test. But I guess it will never change.
Ping @lannka for approval.

**Pull request merged**